pkg/security/tests/schemas/schemas.go

```diff
@@ -89,6 +92,62 @@ func (fi bindataFileInfo) Sys() interface{} {
+
+func bindataChmodSchemaJsonBytes() ([]byte, error) {
+	return bindataRead(
+		_bindataChmodSchemaJson,
+		"/chmod.schema.json",
+	)
+}
+
+func bindataChmodSchemaJson() (*asset, error) {
+	bytes, err := bindataChmodSchemaJsonBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{
+		name:        "/chmod.schema.json",
+		size:        1666,
+		md5checksum: "",
+		mode:        os.FileMode(420),
+		modTime:     time.Unix(1, 0),
+	}
+
+	a := &asset{bytes: bytes, info: info}
+
+	return a, nil
+}
+
+var _bindataChownSchemaJson = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\xe4\x94\x41\x6e\xc3\x20\x10\x45\xf7\x3e\x05\x42\x5d\x26\x21\xf5\xd2\x97\xe8\x01\xaa\xa8\xa2\xf6\xb8\x10\xb9\x40\x61\xda\x2a\x8a\xb8\x7b\x85\x1d\x27\x41\x86\x84\x45\x77\x61\x85\xff\x9f\xc1\x8f\x0f\xe2\x58\x11\x42\x08\x7d\x72\xad\x80\x4f\x4e\x1b\x42\x05\xa2\x71\x0d\x63\x7b\xa7\xd5\x7a\x92\x37\xda\x7e\xb0\xce\xf2\x1e\x59\xbd\xad\xb7\xeb\xe7\x9a\x9d\xea\x57\xa7\x76\xd9\x85\xd6\x56\xe8\x5f\xb5\x09\x8d\xb3\x81\x07\x03\xc1\xd1\xef\x7b\x68\x71\x56\xb9\x3a\xbc\xf4\xb4\x21\xaf\xe3\x67\x18\xc7\xf3\x6c\x5a\xcf\x42\xf0\x69\x2f\x07\x68\x18\x63\xad\x56\xc8\xa5\x02\xfb\x06\x3f\xa0\x70\xfa\xc5\xb9\xc5\xaf\x4a\xd7\x11\xda\x61\x7a\x89\x71\xb6\x9b\xf9\x86\xe1\x36\x9f\x0c\x6e\xac\x8d\xba\xb1\xda\x80\x45\x09\x2e\xe9\x8f\x35\x81\x25\xeb\xe6\x43\x4b\x56\x5a\xf8\xfa\x96\x16\xba\x88\x35\x59\x79\x49\xd0\x70\x14\x34\x5b\xbc\x4b\x3a\x7e\xa1\xc6\x8a\x8f\x19\x29\x0a\x50\xe9\x84\xee\x12\x5f\x48\x97\x90\x31\x9c\x2f\xb8\x00\xb9\x83\xfa\x4f\x8c\xe2\xbd\x3f\xec\xed\x28\x38\xa8\x3b\xd9\xdc\xce\xa5\x2c\x93\xb2\x3c\x68\x07\x0e\xa5\xe2\x28\xaf\xdf\x87\xeb\xb1\x0c\x21\xbb\xdd\xe9\x55\xa9\x7c\xf5\x17\x00\x00\xff\xff\x69\xaa\xbb\xa7\x67\x05\x00\x00")
+
+func bindataChownSchemaJsonBytes() ([]byte, error) {
+	return bindataRead(
+		_bindataChownSchemaJson,
+		"/chown.schema.json",
+	)
+}
+
+func bindataChownSchemaJson() (*asset, error) {
+	bytes, err := bindataChownSchemaJsonBytes()
+	if err != nil {
+		return nil, err
+	}
+
+	info := bindataFileInfo{
+		name:        "/chown.schema.json",
+		size:        1383,
+		md5checksum: "",
+		mode:        os.FileMode(420),
```

Comment:
I think this mode will cause a CI failure. It seems it should be 438 otherwise the CI will report a diff